app/config/parameters.yml.dist

```diff
@@ -41,6 +41,12 @@ parameters:
     paypal.express_checkout.signature: EDITME
     paypal.express_checkout.sandbox: true
 
+    paypal.pro_checkout.username: EDITME
+    paypal.pro_checkout.password: EDITME
+    paypal.pro_checkout.partner: EDITME
+    paypal.pro_checkout.vendor: EDITME
+    paypal.pro_checkout.sandbox: true
```

Comment:
We should not add these parameters here. I know it sucks, but we don't want to have any of these in the parameters.yml.dist, that should be configurable in the backend and it is already possible, but there is no UI for that. Let's add it in our project, but not in Sylius itself. :+1: